Engine/Component/ComponentBillboard.cpp

```diff
@@ -0,0 +1,230 @@
+#include "ComponentBillboard.h"
+
+#include "Main/Application.h"
+#include "Module/ModuleProgram.h"
+#include "Module/ModuleResourceManager.h"
+#include "Module/ModuleRender.h"
+#include "Module/ModuleTime.h"
+
+#include "ResourceManagement/ResourcesDB/CoreResources.h"
+
+ComponentBillboard::ComponentBillboard() : Component(nullptr, ComponentType::BILLBOARD)
+{
+	InitData();
+}
+
+ComponentBillboard::ComponentBillboard(GameObject * _owner) : Component(owner, ComponentType::BILLBOARD)
+{
+	InitData();
+}
+
+ComponentBillboard::~ComponentBillboard()
+{
+	// TODO: Check if not deleting billboard_quad causes a memory leak.
```

Comment:
I think you will need to delete del buffers here, like for example here https://github.com/OnionGalaxy/LittleOrionEngine/blob/develop/Engine/ResourceManagement/Resources/Mesh.cpp#L13

Reply:
right, done!